src/lib.rs

```diff
@@ -80,6 +93,62 @@ impl sdk::Runtime for Runtime {
                     participation_threshold_denominator: 4,
                 },
             },
+            module_contracts::Genesis {
+                parameters: module_contracts::Parameters {
+                    max_code_size: 512 * 1024, // 512 KiB
```

Comment:
Maybe this should be bumped to 1MiB to be safe  (I recall testing some contracts being close to 512KiB already)

Reply:
Done.